addons/pinniped/config-controller/controllers/v1_controller.go

```diff
@@ -54,7 +55,7 @@ func (c *PinnipedV1Controller) SetupWithManager(manager ctrl.Manager) error {
 		).
 		Complete(c)
 	if err != nil {
-		c.Log.Error(err, "error creating pinniped config controller")
+		c.Log.Error(err, fmt.Sprintf("error creating %v", CascadeControllerV1alpha1Name))
```

Comment:
The log actually already includes the name since we set it above, maybe we could just say `error creating controller` here.

Reply:
Ah cool.